kernel/Makefile

```diff
@@ -51,9 +53,12 @@ sign_$(2): build_$(2)
 		 docker tag linuxkit/$(IMAGE):$(1)-$(HASH) linuxkit/$(IMAGE):$(2) && \
 		 DOCKER_CONTENT_TRUST=1 docker push linuxkit/$(IMAGE):$(2))
 
+update-config_$(2): build_$(2)
+	docker run linuxkit/$(IMAGE):$(1)-$(HASH) cat /kernel_config > kernel_config-$(2)
```

Comment:
Yeah, that's why it's `HASH?=` but it still compiles the full kernel, which on my Mac takes a lot longer than just doing the `oldconfig`